components/script/dom/element.rs

```diff
@@ -923,6 +923,10 @@ impl Element {
         &self.local_name
     }
 
+    pub fn tag_name(&self) -> DOMString {
+        self.TagName()
```

Comment:
You should be able to import ElementMethods and invoke TagName() directly from the caller.

Reply:
This public method was added to test something, this will be removed.